assets/src/edit-story/elements/text/edit.js

```diff
@@ -191,11 +200,24 @@ function TextEdit({
     [handleResize]
   );
   // Also invoke if the raw element height ever changes
-  useEffect(handleResize, [elementHeight]);
+  useEffect(handleResize, [
+    elementHeight,
+    font,
```

Comment:
These extra dependencies look wrong to me. The element's height can change independently if it's submitted from the design panel and that'd reflect on the `elementHeight`. If we see some other issues, I'd be even ok if we ran the effect on each render here. So pls let me know what you see.